examples/snippets/notifications/enable_email_notifications.py

```diff
@@ -0,0 +1,53 @@
+"""Example script for generating Email notifications based on Tamr jobs"""
+import tamr_toolbox as tbox
+from tamr_toolbox.models.operation_state import OperationState
+
+# Make Tamr Client
+tamr = tbox.utils.client.create(username="user", password="pw", host="localhost")
+
+# Make Email configuration object that include smtp server information
+# and sender/recipient email addresses
+config = {
+    "sender_address": "sender@gmail.com",
+    "sender_password": "sender_email_password",
+    "recipient_addresses": ["recipient@gmail.com"],
+    "smtp_server": "smtp.gmail.com",
+    "smtp_port": 465,
+}
+
+
+# Send any email message
+tbox.notifications.emails.send_email(
+    message="This is a test message.",
+    subject_line="Subject",
+    sender_address=config["sender_address"],
+    sender_password=config["sender_password"],
+    recipient_addresses=config["recipient_addresses"],
+    smtp_server=config["smtp_server"],
+    smtp_port=config["smtp_port"],
+)
```

Comment:
For this example, it's worth calling out each argument as you are doing, but I want to highlight that because of the way `config` is constructed (matching key names to kwargs) you can do this concisely with
```suggestion
tbox.notifications.emails.send_email(
    message="This is a test message.",
    subject_line="Subject",
    **config,
)
```
If you had a more general config object, my recommendation would be to have your current one as a dictionary under `"mail_config"`, and then pass `**config["mail_config"]`.

Reply:
That is very smart! Will use in the future